lib/sqs.js

```diff
@@ -4,6 +4,27 @@ const _ = require('lodash');
 const AWS = require('aws-sdk');
 const Promise = require('bluebird');
 const clientConfig = require('./client-config');
+const uuid = require('uuid/v4');
+const util = require('util');
+const zlib = require('zlib');
+const compress = util.promisify(zlib.gzip);
```

Comment:
i know @pmcclory was talking of other compression algorithms. I guess we'll go out the gate with this and course correct if need be?

Reply:
👍 yeah I slightly investigated this and then fell back to gzip.  the results were promising for facebook's `zstd` lib but the nodejs module is kinda outdated (and it won't work on node 10, i tried it out), so this seemed like a safer starting point